gdcdictionary/schemas/molecular_test.yaml

```diff
@@ -6996,8 +6996,11 @@ properties:
       - mcg/mL
```

Comment:
I check the prod db for existing `test_units` and it looks like there's a value that's not in this enum list.  specifically, `count x10^9/L`.  Should that value be added or migrated?

For reference, here's the list of value and the number of occurrences:
```
In [4]: collections.Counter(mt.test_units for mt in g.nodes(MolecularTest).yield_per(100))                                                                                                                                         
Out[4]: 
Counter({'mg/dL': 15687,
         'count x10^9/L': 25195,
         'mmol/L': 32619,
         'ukat/L': 5512,
         None: 15030,
         'ng/mL': 7,
         'mcg/mL': 3187,
         'g/L': 28996,
         'umol/L': 8409,
         'g/dL': 15376})
```

Reply:
Should be migration, see [ticket](https://jira.opensciencedatacloud.org/browse/DAT-3063)

Reply:
Thank you @philtom-ctds. I updated the deprecated enum to match the missed enum value (typo error)